Formula/icecast.rb

```diff
@@ -1,8 +1,10 @@
+require "English"
```

Comment:
Why?

Reply:
The homebrew auditor yells at you when you use $? so you have to use $CHILD_STATUS, which requires the "English" module (for some reason)

Reply:
```suggestion
```

In that case, please remove it here, and see my comments below

Reply:
Also, FWIW, I tested on my system and I did not beed "English" with your formula. Maybe something is fishy with your ruby env?